src/Libraries/RevitNodes/Elements/Floor.cs

```diff
@@ -70,13 +70,18 @@ private void InitFloor(Autodesk.Revit.DB.Floor floor)
         /// <summary>
         /// Initialize a floor element
         /// </summary>
-        private void InitFloor(List<CurveLoop> profiles, Autodesk.Revit.DB.FloorType floorType, Autodesk.Revit.DB.Level level)
+        private void InitFloor(List<CurveLoop> profiles, Autodesk.Revit.DB.FloorType floorType, Autodesk.Revit.DB.Level level, double offset = 0)
         {
             TransactionManager.Instance.EnsureInTransaction(Document);
 
             // we assume the floor is not structural here, this may be a bad assumption
             Autodesk.Revit.DB.Floor floor = Autodesk.Revit.DB.Floor.Create(Document, profiles, floorType.Id, level.Id);
-
+            var param = floor.get_Parameter(BuiltInParameter.FLOOR_HEIGHTABOVELEVEL_PARAM);
+            var paramName = param.ToString();
+            if(param.AsDouble() == 0 && offset != 0)
```

Comment:
Do you mean `if (param.AsDouble() != offset)` ? And where do we use the variable `paramName`?

Reply:
According to the new Revit API, this parameter of new Floor should be 0. May be we should add an exception for it?
the "paramName" used to get and set Element's parameter, but I found that ElementUtils can be used directly. So it should be removed.